docs/Ingress.md

```diff
@@ -543,6 +551,59 @@ spec:
           servicePort: 8000
 ```
 
+Example to create a TLS enabled Ingress configuration
```

Comment:
It's best not to leave any prerequisites, notes, conditions, or other gotchas at the bottom. 

Example to create a TLS enabled Ingress configuration
There are a few important configurations to be aware of in the TLS configuration:
* The `nginx.ingress.kubernetes.io/backend-protocol:` requires `"HTTPS"` when TLS is configured on the backend services.
* The `secretName` must reference a [valid TLS secret](https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets).

Note: This example assumes that https is enabled for Splunk Web.

Reply:
Got it, will update.